### ISSUE TITLE
The link color is probably not A11y compliant

### ISSUE THREAD

**Issue by dgading** [dgading/blogfolio_frontend#12]
The pewterblue color needs to be darkened in posts and projects as it is probably doesn't meet accessibility requirements.
